kernel/mount.c

```diff
@@ -39,13 +43,17 @@ static size_t _mount_table_size = 0;
 static myst_spinlock_t _lock = MYST_SPINLOCK_INITIALIZER;
 
 static bool _installed_free_mount_table = false;
+static bool _created_automount_dir = false;
```

Comment:
maybe drop this flag and just check for the existence of the directory each time like this.

```
struct stat statbuf;

if (!(myst_syscall_stat(pathname, &statbuf) && S_ISDIR(statbuf.st_mode)))
{
}
```

Reply:
Done.